tests/e2e/logging_test.go

```diff
@@ -37,15 +37,22 @@ func TestNoErrorLogsDuringNormalOperations(t *testing.T) {
 				e2e.WithClusterSize(1),
 				e2e.WithLogLevel("debug"),
 			},
-			allowedErrors: map[string]bool{"setting up serving from embedded etcd failed.": true},
+			allowedErrors: map[string]bool{
+				"setting up serving from embedded etcd failed.": true,
+			},
```

Comment:
Why don't we ignore the error in such case? Suggest to add it into this case as well.

Reply:
We can ignore it here as well, but according to testgrid there was no failure for this test. Only the three node cases were affected from what I can see.

Reply:
In theory, one member may still run into the issue, but just with a low possibility.

Reply:
Ok, I added the exclusion for this test as well.